### PR TITLE
Update NonAccMachineSGXLinuxGettingStarted.md

### PR DESCRIPTION
**Pull request by RokasAidukas-Centric**
Changed PCCS server's test URL from v2 to v3. V2 no longer works.

### CODE REVIEW

**Review by achamayou — APPROVED** [review]

**Review by radhikaj — APPROVED** [review]
bors r+

**Comment by asvrada** [comment]
Hi @RokasAidukas-Centric , the commit is missing signoff so we can't merge it. Do you mind run `git commit --amend -s` and `git push -f` to update this PR?

**Comment by RokasAidukas-Centric** [comment]
@asvrada Done!

**Pull request merged**
